### PR TITLE
feat(logging): less verbose output

### LINKED ISSUE #2088: The push step (3) is overly verbose
I've recently set up a new Earthly project. I execute `earthly +build` and it outputs
```
$ earthly +build
...
   +generate-graphql | --> SAVE ARTIFACT graphql/model/models_gen.go +generate-graphql/models_gen.go AS LOCAL graphql/model/models_gen.go
              output | --> exporting outputs
              output | [██████████] 100% copying files
              output |

 3. Push ⏫ (disabled)
————————————————————————————————————————————————————————————————————————————————

To enable pushing use

		earthly --push ...



 4. Local Output 🎁
————————————————————————————————————————————————————————————————————————————————

Artifact +generate-graphql/generated output as graphql/generated
Artifact +generate-graphql/models_gen.go output as graphql/model/models_gen.go


========================== 🌍 Earthly Build  ✅ SUCCESS ==========================

Share your logs with an Earthly account (experimental)! Register for one at https://ci.earthly.dev.
```
Every time I execute this I need to glance past the overly verbose
```

 3. Push ⏫ (disabled)
————————————————————————————————————————————————————————————————————————————————

To enable pushing use

		earthly --push ...


```
section to get to my compilation errors. I assume the information is there to inform me that I can push Docker images using Earthly, but really right now, and possibly never for this project, am I not interested in this. Some things that would make the output nicer:
 * If I could somehow disable/hide the push output. (best)
 * If the push information was less verbose and more compact (better).

### PR DESCRIPTION
**Pull request by JensRantil**
...for "earthly --push ...".

This fixes #2088.

### CODE REVIEW

**Comment by JensRantil** [comment]
Unfortunately I was unable to test this change locally. The error I got was:
```
$ ~/src/earthly/build/darwin/amd64/earthly +test-formats

 1. Init 🚀
————————————————————————————————————————————————————————————————————————————————

           buildkitd | Starting buildkit daemon as a docker container (earthly-buildkitd)...
Share your logs with an Earthly account (experimental)! Register for one at https://ci.earthly.dev.
Error: build new buildkitd client: maybe start buildkitd: wait until started: buildkitd crashed
It seems that buildkitd is shutting down or it has crashed. You can report crashes at https://github.com/earthly/earthly/issues/new.

================================= System Info ==================================

version: dev-main
build-sha: cc84828a953fb75db68c3f0656e42afbbbd980b3
platform: unknown

================================ Docker Version ================================

&containerutil.FrontendInfo{ClientVersion:"20.10.11", ClientAPIVersion:"1.41", ClientPlatform:"darwin/arm64", ServerVersion:"20.10.11", ServerAPIVersion:"1.41", ServerPlatform:"linux/arm64", ServerAddress:"/var/run/docker.sock"}

================================ Buildkit Logs =================================

starting earthly-buildkit with EARTHLY_GIT_HASH=cc84828a953fb75db68c3f0656e42afbbbd980b3 BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:3b049bfd9c25598450037d0911255f9e3ba8d0ed+build
detected cgroups v2; buildkit/entrypoint.sh running under pid=1
Autodetecting iptables
Could not find an ip_tables module; falling back to heuristics.
Detected iptables-legacy by output length (0 >= 0)
BUILDKIT_ROOT_DIR=/tmp/earthly/buildkit
CACHE_SIZE_MB=0
BUILDKIT_MAX_PARALLELISM=20
BUILDKIT_LOCAL_REGISTRY_LISTEN_PORT=8371
EARTHLY_ADDITIONAL_BUILDKIT_CONFIG=
CNI_MTU=1500

======== CNI config ==========
{
	"cniVersion": "0.3.0",
	"name": "buildkitbuild",
	"type": "bridge",
	"bridge": "cni0",
	"isGateway": true,
	"ipMasq": true,
	"mtu": 1500,
	"ipam": {
		"type": "host-local",
		"subnet": "172.30.0.0/16",
		"routes": [
			{ "dst": "0.0.0.0/0" }
		]
	}
}
======== End CNI config ==========

======== Buildkitd config ==========
debug = false
root = "/tmp/earthly/buildkit"
insecure-entitlements = [ "security.insecure" ]





[worker.oci]
  enabled = true
  snapshotter = "auto"
  max-parallelism = 20
  gc = true
  networkMode = "cni"
  cniBinaryPath = "/usr/libexec/cni"
  cniConfigPath = "/etc/cni/cni-conf.json"



======== End buildkitd config ==========
Detected container architecture is x86_64
starting shellrepeater
```

Let me know if there is an easy fix. Otherwise, feel free to test locally and merge if the output looks good.

**Comment by JensRantil** [comment]
Running
```
$ earthly --version
earthly version v0.6.21 7e4f1df4c124db1644d51d312b19313217cbe478 darwin/arm64; macOS 12.4
```
locally works without any issues.

**Comment by vladaionescu** [comment]
> The error I got was

What was the command you used to build Earthly itself?

File: builder/builder.go [hunk 1]
```diff
@@ -446,7 +446,7 @@ func (b *Builder) convertAndBuild(ctx context.Context, target domain.Target, opt
 	if opt.PrintPhases {
 		b.opt.Console.PrintPhaseHeader(PhasePush, !opt.Push, "")
 		if !opt.Push {
-			b.opt.Console.Printf("To enable pushing use\n\n\t\tearthly --push ...\n\n")
+			b.opt.Console.Printf("To enable pushing use `earthly --push ...`.\n")
```
Comment (vladaionescu) [inline builder/builder.go:449]:
```suggestion
			b.opt.Console.Printf("To enable pushing use earthly --push\n")
```

**Comment by JensRantil** [comment]
> > The error I got was
> 
> What was the command you used to build Earthly itself?

`./earthly +for-darwin-m1` based on instructions in https://github.com/earthly/earthly/blob/main/CONTRIBUTING.md#building-from-source

**Comment by vladaionescu** [comment]
> `./earthly +for-darwin-m1` based on instructions in ...

Yeah ok - that looks right. Curious what the output of `docker logs earthly-buildkitd` is when this happens?

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**